### PR TITLE
updated readme with installation instructions

### PR DESCRIPTION
**Pull request by david-piiano**

### CODE REVIEW

File: aws-apprunner-nodejs-demo-app/README.md [hunk 1]
```diff
@@ -6,15 +6,69 @@ The sample node application has been built on top of AWS AppRunner using typescr
...
+   ![Step2](./resources/AppRunner-step2.png)
+
+1. Settings for step 3: `Configure service`.  
+   Don't forget to **set the same VPC as the Vault** in the networking section.
+
+   ![Step3](./resources/AppRunner-step3.png)
+
+1. Setting for step 3: `Health check`
+
+   ![Step3](./resources/AppRunner-step3-health.png)
```
Comment (ori-shalom) [inline aws-apprunner-nodejs-demo-app/README.md:49]: Did you test it? Does it really need port 3000?
I think it's a mistake here and in the other curl commands below

**Review by ori-shalom — APPROVED** [review]
Approved with comments.
Notice I reviewed from my phone so I might have missed few things but I don't want to block you and docs can always be fixed later...

File: aws-apprunner-nodejs-demo-app/README.md [hunk 2]
```diff
@@ -6,15 +6,52 @@ The sample node application has been built on top of AWS AppRunner using typescr
...
+You must have the [AppRunner](../aws-apprunner) installed already and the details of the application URL and the authentication token available.
+This application uses the Vault typescript SDK version 1.0.8.
+
+## Steps
+
+1. Clone this repository
+1. Move this directory into a new repository of its own. Let's call it `demo-app` under your account `myaccount`. It is assumed that the repository will be hosted in: `https://github.com/myaccount/demo-app`
+1. Modify [apprunner.yaml](apprunner.yaml):
+   1. Change `PVAULT_URL` containing generic xxxx. to the right URL of the Vault
+   1. Change `PVAULT_API_KEY` account number from the generic `123456789012` into your account number
```
[the remark this answers is not in the document]
Comment (david-piiano) [reply aws-apprunner-nodejs-demo-app/README.md:20]: Right, I'll clarify. The key contains the account number

File: aws-apprunner-nodejs-demo-app/README.md [hunk 3]
```diff
@@ -6,15 +6,69 @@ The sample node application has been built on top of AWS AppRunner using typescr
...
+1. Open the AWS Console in the AppRunner section: https://us-east-2.console.aws.amazon.com/apprunner
```
[the remark this answers is not in the document]
Comment (david-piiano) [reply aws-apprunner-nodejs-demo-app/README.md:21]: To make the link work - I'll clarify

Comment (david-piiano) [inline aws-apprunner-nodejs-demo-app/README.md:21]:
```suggestion
1. Open the AWS Console in the AppRunner section. For example, for `us-east-2`: https://us-east-2.console.aws.amazon.com/apprunner
```

File: aws-apprunner-nodejs-demo-app/README.md [hunk 4]
```diff
@@ -6,15 +6,69 @@ The sample node application has been built on top of AWS AppRunner using typescr
...
+
+      ![step1c](./resources/AppRunner-connect-to-github.png)
+
+   1. Select automatic mode as the `Deployment settings`
+
+1. Select to use a [configuration file](apprunner.yaml) in the `build settings`.
+
+   ![Step2](./resources/AppRunner-step2.png)
+
+1. Settings for step 3: `Configure service`.
```
[the remark this answers is not in the document]
Comment (david-piiano) [reply aws-apprunner-nodejs-demo-app/README.md:42]: Yes, just checked it. My bad. Fixing

File: aws-apprunner-nodejs-demo-app/README.md [hunk 5]
```diff
@@ -6,15 +6,69 @@ The sample node application has been built on top of AWS AppRunner using typescr
...
+
+1. Setting for step 3: `Security`
+
+   ![Step3](./resources/AppRunner-step3-security.png)
+
+1. Deploy!
 
 ## Usage
 
 Once deployed this application exposes port 3000 on the AppRunner URL for calls from the internet.
```
Comment (david-piiano) [inline aws-apprunner-nodejs-demo-app/README.md:59]:
```suggestion
Once deployed this application runs over HTTPS (443) on the AppRunner URL for calls from the internet.
```

File: aws-apprunner-nodejs-demo-app/README.md [hunk 6]
```diff
@@ -6,15 +6,69 @@ The sample node application has been built on top of AWS AppRunner using typescr
...
+
+The demo application connects to the Vault (which is also running in AppRunner) and retrieves the status and version. Note that the version requires correct authentication configuration. If it ran successfully, it means all is configured well.
+
 Example:
 
 ```
-curl <app runner endpoint>:3000
+> curl https://<your magic prefix>.us-east-2.awsapprunner.com:3000
```
Comment (david-piiano) [inline aws-apprunner-nodejs-demo-app/README.md:66]:
```suggestion
> curl https://<your magic prefix>.us-east-2.awsapprunner.com```

**Pull request merged**